api/job.go

```diff
@@ -559,7 +559,7 @@ func deleteJob(w http.ResponseWriter, r *http.Request, t auth.Token) (err error)
 	}
 	if err = servicemanager.Job.RemoveJob(ctx, j); err != nil {
 		if err == jobTypes.ErrJobNotFound {
```

Comment:
@crgisch there are some place to inform user that put image on create was deprecated?